### ISSUE TITLE
Add part of Cheek Pouch Content, e.g. food items found in heteromyid rodent cheek pouches

### ISSUE THREAD

**Issue by cjconroy** [ArctosDB/arctos#3153]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Goal**
Add a part that will be searchable for contents of mammal cheek pouches, like seeds. 

**Context**
The MVZ has a collection of these contents that go along with cataloged specimens. They have never been curated an any organized way, and these parts have not been added to Arctos, even as remarks. Adding as a part will allow them to be searchable, and curated. Additional notes on classification could be added. Some may be referenced in publications. 
I am unaware if other mammal collections have items like this, but this might benefit other museums. 

Arctos has the part of cheek pouch, and cheek content (dry). Cheek pouch is used only once in Arctos, for a specimen at MSB. It is being researched, but is unclear whether this is the pouch, the contents, or both. 
https://arctos.database.museum/guid/MSB:Mamm:269922

Cheek content is used only twice at MVZ for material from a bird mouth. I am not sure this is the same function as this material may have been food or nesting material. 
Here is an example. 
https://arctos.database.museum/guid/MVZ:Bird:60376

Attached are images of some contents from kangaroo rats. 

**Value**
Cheek pouch contents

**Definition**
Typically, food items removed from the cheek pouches of mammals including the platypus, rodents like cricetids, sciurids, heteromyids and geomyids, and most monkeys, as well as the marsupial koala.

**Collection type**
Mamm

**Attribute data type**
free-text

**Priority**
Medium - needs discussion from other mammal collections
![Dixon 6698](https://user-images.githubusercontent.com/5749672/94872552-bef2e580-0401-11eb-8311-671287349cb6.jpg)
![Swarth 9135](https://user-images.githubusercontent.com/5749672/94872567-c4503000-0401-11eb-8166-5f3cdc410679.jpg)

**Comment by dustymc** [comment]
I certainly support cataloging this in some usable way, but I'm not sure if we should have "{every possible cavity} contents" parts or perhaps if these can/should be merged into "consumed items" or something of the sort. Here's existing data.

```
arctosprod@arctos>> select part_name from ctspecimen_part_name where part_name like '%content%' group by part_name order by part_name;
                 part_name                  
--------------------------------------------
 bill content (dry)
 cheek content (dry)
 crop content (70% ethanol)
 crop content (dry)
 egg contents (frozen)
 hindgut content (70% ethanol)
 hindgut content (ethanol)
 stomach content
 stomach content (70% ethanol)
 stomach content (95% ethanol)
 stomach content (bouins fluid)
 stomach content (dry)
 stomach content (ethanol)
 stomach content (ethanol-fixed)
 stomach content (formalin)
 stomach content (frozen)
 stomach content (isopropanol)
 stomach content (RNAlater)
 stomach plus crop content (95% ethanol)
 stomach plus crop content (dry)
 stomach plus stomach content (70% ethanol)
 stomach plus stomach content (95% ethanol)
```

**Comment by ewommack** [comment]
Part of the problem may be that cheek content (dry) is only available for birds and I think @cjconroy may need it for mammals.

**Comment by dustymc** [comment]
> available for birds

If that's suitable then I vote to proceed with this, and maybe have the big-picture discussion later.

**Comment by cjconroy** [comment]
I have a cheek, but I don't have a cheek pouch like a kangaroo rat. I also don't have whatever a Ptarmigan has. But, the part  cheek content (dry) could suffice for now until more mammalogists chime in.

**Comment by Jegelewicz** [comment]
Added to Mamm collections

**Comment by cjconroy** [comment]
Hi,
     I can't add part of Cheek Content for mammals. I don't think it was added to mamm parts options. Can that be done?  Thanks. 
<img width="803" alt="Screen Shot 2022-09-12 at 10 30 57 AM" src="https://user-images.githubusercontent.com/5749672/189726600-cac2f2f3-e685-49fd-855f-21af5ec575f2.png">

**Comment by Jegelewicz** [comment]
added
